### PR TITLE
Adds functions_helloworld_http sample

### PR DESCRIPTION
**Pull request by grant**
Adds functions_helloworld_http in addition to functions_helloworld_get. (Not sure why we need both)
b/132732935

### CODE REVIEW

File: functions/snippets/src/main/java/HelloWorld.java [hunk 1]
```diff
@@ -14,19 +14,37 @@
...
+// [START functions_helloworld_http]
 // [START functions_helloworld_get]
 import java.io.IOException;
 import java.io.PrintWriter;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
 public class HelloWorld {
+  // [END functions_helloworld_get]
+  // [START functions_helloworld_http]
```
Comment (lesv) [inline functions/snippets/src/main/java/HelloWorld.java]: Your missing an END of this from L17

File: functions/snippets/src/test/java/SnippetsTests.java [hunk 2]
```diff
@@ -81,11 +81,18 @@ public void afterTest() {
 
   @Test
   public void helloWorldTest() throws IOException {
-    new HelloWorld().helloGet(request, response);
+    new HelloWorld().helloWorld(request, response);
```
Comment (lesv) [inline functions/snippets/src/test/java/SnippetsTests.java]: Looks like this change was wrong.

File: functions/snippets/src/main/java/HelloWorld.java [hunk 3]
```diff
@@ -14,19 +14,37 @@
...
  */
 
+// [START functions_helloworld_http]
 // [START functions_helloworld_get]
 import java.io.IOException;
 import java.io.PrintWriter;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
 public class HelloWorld {
```
Comment (eamonnmcmanus) [inline functions/snippets/src/main/java/HelloWorld.java]: This should use the new API (i.e., implement HttpFunction).

**Comment by lesv** [comment]
Should we close this PR?

**Comment by grant** [comment]
There have been lots of updates to the Java Functions Framework since the creation of this PR on Jan 22. I haven't focused on this language as the FF has been changing.
https://github.com/GoogleCloudPlatform/functions-framework-java

Thanks for all the comments and linking the `SAMPLE_FORMAT.md` doc. A new PR would be better to continue adding these snippets. Thus closing in favor of a new PR.

**Pull request closed without merging**